conda/meta.yaml

```diff
@@ -33,6 +33,7 @@ test:
     - ts-dds
     - ts-idl
     - ts-salobj
+    - ts-simactuators
     - black
   source_files:
     - python
```

Comment:
Please also add `ts-simactuators` and `ts-idl` as a "run" dependency.

Reply:
Thanks for catching that.